tests/model_inheritance/tests.py

```diff
@@ -184,13 +184,11 @@ def b():
             GrandChild().save()
 
         for i, test in enumerate([a, b]):
-            with self.subTest(i=i), self.assertNumQueries(4), CaptureQueriesContext(
-                connection
-            ) as queries:
+            with self.subTest(i=i), CaptureQueriesContext(connection) as queries:
                 test()
                 for query in queries:
                     sql = query["sql"]
-                    self.assertIn("INSERT INTO", sql, sql)
+                    self.assertNotIn("UPDATE ", sql, sql)
```

Comment:
This changes the meaning of the test as it no longer proves that INSERTs were executed.